docs/user/tutorial/3_viz.rst

```diff
@@ -0,0 +1,642 @@
+    (B, A, C)    575
+    (A, C, B)    219
+    (C, A, B)    131
+    (B, C, A)     28
+    (A, B, C)     26
+    (C, B, A)     21
+
+
+
+.. image:: 3_viz_files/3_viz_22_1.png
+
+
+.. parsed-literal::
+
+    (1,) {'weight': 0, 'cast': False}
+    (1, 2, 3) {'weight': Fraction(28, 1), 'cast': True}
+    (1, 3, 2) {'weight': Fraction(575, 1), 'cast': True}
+    (2,) {'weight': 0, 'cast': False}
+    (2, 3, 1) {'weight': Fraction(131, 1), 'cast': True}
+    (2, 1, 3) {'weight': Fraction(21, 1), 'cast': True}
+    (3,) {'weight': 0, 'cast': False}
+    (3, 1, 2) {'weight': Fraction(26, 1), 'cast': True}
+    (3, 2, 1) {'weight': Fraction(219, 1), 'cast': True}
+
+
+Check that this is reasonable: only ballots that were in the
+``PreferenceProfile`` should have ``cast = True``, and their ``weight``
+attribute should correspond to the number of ballots cast. Why do none
+of the bullet votes appear in the profile?
+
+**Try it yourself**
+~~~~~~~~~~~~~~~~~~~
+
+   If we wanted to visualize only the nodes corresponding to cast
+   ballots, we use the ``show_cast = True`` parameter in the ``draw``
+   method. You can go back and try that above.
+
+What if we wanted to explore a particular neighborhood of a ballot?
+Let’s look at the radius-1 neighborhood around the ballot (3,2,1,4).
+This is also called the *1-neighborhood*, and it means (3,2,1,4) and its
+immediate neighbors, with their interconnections shown. The
+0-neighborhood is only a point itself; the 2-neighborhood is everything
+within two steps on the ballot graph.
+
+Here we will initialize the ballot graph from a number, representing the
+number of candidates.
+
+.. code:: ipython3
+
+    ballot_graph = BallotGraph(4)
+    ballot_graph.draw()
+    
+    # the neighborhoods parameter takes a list of tuples (node, radius)
+    # and displays the corresponding neighborhoods
+    ballot_graph.draw(neighborhoods=[((3,2,1,4), 1)])
+
+
+
+.. image:: 3_viz_files/3_viz_25_0.png
+
+
+
+.. image:: 3_viz_files/3_viz_25_1.png
+
+
+We can also draw multiple neighborhoods.
+
+**Try it yourself**
+~~~~~~~~~~~~~~~~~~~
+
+   In addition to the 1-neighborhood of (3,2,1,4), draw the
+   1-neighborhood of (2,). Note that you have to write (2,) and not
+   simply (2) to designate the node with a bullet vote for candidate 2.
+
+Scottish Elections
+------------------
+
+Scottish elections give us a great source for real-world ranked data,
+because STV is used for local government elections. Thanks to `David
+McCune <https://www.jewell.edu/faculty/david-mccune>`__ of William
+Jewell College, we have a fantastic
+`repository <https://github.com/mggg/scot-elex>`__ of shiny, clean
+ranking data from over 1000 elections, which feature 3-14 candidates
+apiece, running with a party label.
+
+Here we load in the CVR from a ward in Comhairle nan Eilean Siar in
+2012, in the election for city council. Please download the csv file
+`here <https://github.com/mggg/scot-elex/blob/main/4_cands/eilean_siar_2012_ward3.csv>`__
```

Comment:
I've been told it is called a BLT file!

Reply:
That's mildy, absolutley fantastic. Is there a way to change the file extension so that it says ".blt" rather than ".csv" so people don't try to open it in excel?

Reply:
I'm not sure; I haven't done any managing of that repo. I'm sure I could rig up a bash script to change them all. Let me ping the votekit channel.

Reply:
Alright so the plan will be to actually change the files to be csv's.